pages/linux/hashcat.md

```diff
@@ -0,0 +1,32 @@
+# hashcat
+
+> Fast and advanced password recovery tool.
+> More information: <https://hashcat.net/hashcat/>.
+
+- Brute-force attack with default hashcat mask:
```

Comment:
```suggestion
- Perform a brute-force attack with default hashcat mask (`hash[:salt]:plain`):
```
Same with the all the other examples except the last.
I'm not sure if the second part of the suggestion is correct, but we should at least somewhere explain what `--attack-mode {{3}}` means.

Reply:
I do get what you mean with 2 (`hash[:salt]:plain`), could you explain further? 

The default hashcat mask is "?1?2?2?2?2?2?2?3?3?3?3?d?d?d?d" and this works incrementally (from 1 chars to 15). More info can be found here https://hashcat.net/wiki/doku.php?id=hashcat#default_values but I have not found a way to explain this in one line

Reply:
About the --attack-mode
 # | Mode
 ===+======
  0 | Straight
  1 | Combination
  3 | Brute-force
  6 | Hybrid Wordlist + Mask
  7 | Hybrid Mask + Wordlist

Because the example  explains specifically how to do a  "brute-force attack" (option 3) I think set it as a variable is incorrect, indeed "--attack-mode" is not a "variable" of the burte-force mode attack , instead {{hash_value}} is. What do you think? 

The sentence "Perform a brute-force attack (mode 3) with default hashcat mask" , would work for you?

Reply:
> I do get what you mean with (`hash[:salt]:plain`), could you explain further?

I didn't know that `mode 3` referred to the attack mode. I thought it was referring to the outfile format mode 3, which is `hash[:salt]:plain`. 

> The sentence "Perform a brute-force attack (mode 3) with default hashcat mask" would work for you?

Sounds good!